VisualRust.Build/Rustc.cs

```diff
@@ -182,8 +192,53 @@ public override bool Execute()
             }
         }
 
+        private bool FindRustc()
+        {
+            string target = TargetTriple ?? Shared.Environment.DefaultTarget;
+            installPath = Shared.Environment.FindInstallPath(target);
+            if (installPath == null)
+            {
+                if (String.Equals(target, Shared.Environment.DefaultTarget, StringComparison.OrdinalIgnoreCase))
+                    Log.LogError("No Rust installation detected. You can download official Rust installer from https://www.rust-lang.org/downloads.html");
+                else
+                    Log.LogError("Could not find a Rust installation that can compile target {0}.", target);
+                return false;
+            }
+
+            return true;
+        }
+
+        private Process CreateProcess(String argumets)
+        {
+            var psi = new ProcessStartInfo()
+            {
+                CreateNoWindow = true,
+                FileName = Path.Combine(installPath, "rustc.exe"),
+                UseShellExecute = false,
+                WorkingDirectory = WorkingDirectory,
+                Arguments = argumets,
+                RedirectStandardError = true,
+                RedirectStandardOutput = true
+            };
+
+            var process = new Process();
+            process.StartInfo = psi;
+
+            return process;
+        }
+
+        private Version GetVersion()
+        {
+            var process = CreateProcess(" --version");
+            process.Start();
+            process.WaitForExit();
+            return Version.ParseRustcOutput(process.StandardOutput.ReadToEnd());
+        }
+
         private bool ExecuteInner()
         {
+            var useJsonErrorFormat = rustcVersion.VersionMinor >= 11;
```

Comment:
Can you also check `rustcVersion.VersionMajor >= 1` ?  I don't think that anyone uses v0.11 anymore, but the check just seems ... incomplete.